odoo/addons/test_assetsbundle/tests/test_js_transpiler.py

```diff
@@ -255,7 +255,10 @@ def test_06_import(self):
 import {Line13}  from "@test.Dialog" //HELLO
 
 
-const test = `import { Line13, Notification } from "../src/Dialog";`
+const test = `import { Line14, Notification } from "../src/Dialog";`
+
+import Line15 from "test/Dialog";
+import Line16 from "test.Dialog.error";
```

Comment:
Could the commit message explain more clearly why / how these differ from cases 7/8/9?

Reply:
15 and 16 are legacy js file imports.
7/8 and 9 are @odoo-module js file imports